demo/mobile/src/app/initialisation/config.js

```diff
@@ -16,16 +16,14 @@ angular.module('wfm-mobile').config(['$stateProvider', '$urlRouterProvider', cre
   '$rootScope', '$scope', '$state', '$mdSidenav', 'userService', '$mdDialog',
   function($rootScope, $scope, $state, $mdSidenav, userService, $mdDialog) {
     userService.readUser().then(function(profileData) {
-      $scope.profileData = profileData;
+      if (profileData) {
+        $scope.profileData = profileData;
+      } else {
+        userService.login();
```

Comment:
This should never happen. After login we should save profile data and never remove it. 
Code may be good for cases when someone remove local storage when testing :D

Reply:
That would only get called when the user logs out.